packages/components/src/internal/components/forms/QueryFormInputs.tsx

```diff
@@ -288,15 +288,18 @@ export class QueryFormInputs extends React.Component<QueryFormInputsProps, State
                     } else if (col.inputType === 'file' && renderFileInputs) {
                         return (
                             <FileInput
+                                formsy
                                 key={i}
                                 queryColumn={col}
                                 value={value}
-                                onChange={onChange}
+                                onChange={onFileChange}
+                                name={col.fieldKey}
                                 allowDisable={allowFieldDisable}
                                 initiallyDisabled={shouldDisableField}
                                 onToggleDisable={this.onToggleDisable}
                                 addLabelAsterisk={showAsteriskSymbol}
                                 renderFieldLabel={renderFieldLabel}
+                                showLabel
```

Comment:
So `FileInput` wasn't previously showing the label? or did you change the default prop there?

Reply:
Yeah, the layout of the label for this field was wonky due to this. It was still rendering a label but not adhering to the same layout specifications as our other form inputs.